### ISSUE TITLE
The fr_weight and header_weight parameters

### ISSUE THREAD

**Issue by vietthanh2710** [oufuzhao/CLIB-FIQA#4]
Can you please give the fr_weight and header_weight parameters, I found it hard to obtain those weights from the training instruction of CR-FIQA. 
Thanks a lot.

**Comment by oufuzhao** [comment]
Hi @vietthanh2710, thanks for your interest. 

Unfortunately, I may not be able to find fr_weight and header_weight anymore. And, considering that the identity label (class index) of header_weight and ID in the dataset may be different in different training sets, we suggest re-obtain it in your dataset using CR-FIQA. 

In fact, for fr_weight and header_weight, you just save need to use torch.save() to save the [backbone](https://github.com/fdbtrs/CR-FIQA/blob/51d38952c8f606f1454cf46f90af44de3227a6ce/train_cr_fiqa.py#L149) and [header](https://github.com/fdbtrs/CR-FIQA/blob/51d38952c8f606f1454cf46f90af44de3227a6ce/train_cr_fiqa.py#L150) in the training code of CR-FIQA:

Backbone_SAVE_PATH = "/XXXXX/XXXX/Your_Path_ fr_weight.pth"
Header_SAVE_PATH = "XXXXX/XXXX/Your_Path_ header_weight.pth"
torch.save(backbone.state_dict(), Backbone_SAVE_PATH)
torch.save(header.state_dict(), Header_SAVE_PATH)
